codeSnippets/snippets/tutorial-http-api/build.gradle.kts

```diff
@@ -19,10 +19,10 @@ repositories {
 }
 
 dependencies {
-    implementation("io.ktor:ktor-server-core:$ktor_version")
-    implementation("io.ktor:ktor-server-netty:$ktor_version")
-    implementation("io.ktor:ktor-server-content-negotiation:$ktor_version")
-    implementation("io.ktor:ktor-serialization-kotlinx-json:$ktor_version")
+    implementation("io.ktor:ktor-server-core-jvm")
```

Comment:
Did we already migrate to the BOM in the documentation (the `$ktor_version` is removed)?

Reply:
What do you mean by "migrate to the BOM"? I only replaced the dependency names to match the ones the generator adds (and in version `2.3.6` it adds them without the `$ktor_version`, but with the `-jvm` suffix instead)